packages/smooth_app/lib/main.dart

```diff
@@ -68,35 +109,18 @@ class _SmoothAppState extends State<SmoothApp> {
   @override
   void initState() {
     super.initState();
-    _initFuture = _init();
+    _initFuture = _init2();
   }
 
-  Future<void> _init() async {
+  Future<bool> _init2() async {
+    await _init1();
     systemDarkmodeOn = brightness == Brightness.dark;
-    _userPreferences = await UserPreferences.getUserPreferences();
-    _localDatabase = await LocalDatabase.getLocalDatabase();
-    _productPreferences = ProductPreferences(
-      ProductPreferencesSelection(
-        setImportance: _userPreferences.setImportance,
-        getImportance: _userPreferences.getImportance,
-        notify: () => _productPreferences.notifyListeners(),
-      ),
-      daoString: DaoString(_localDatabase),
-    );
-
     await _productPreferences.init(DefaultAssetBundle.of(context));
-    await _userPreferences.init(_productPreferences);
-
-    AnalyticsHelper.setCrashReports(_userPreferences.crashReports);
-    AnalyticsHelper.setAnalyticsReports(_userPreferences.analyticsReports);
-    ProductQuery.setCountry(_userPreferences.userCountryCode);
-    _themeProvider = ThemeProvider(_userPreferences);
-    ProductQuery.setQueryType(_userPreferences);
-
-    cameras = await availableCameras();
-
-    await ProductQuery.setUuid(_localDatabase);
-    AnalyticsHelper.initMatomo(context);
+    if (!_screenshots) {
+      await _userPreferences.init(_productPreferences);
+    }
+    AnalyticsHelper.initMatomo(context, _screenshots);
```

Comment:
We don't need _screenshots as argument as it's a global variable

Reply:
I don't think it is: it's private.